tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/FindReplaceDialogTest.java

```diff
@@ -426,6 +429,20 @@ public void testActivateWholeWordsAndSearchForTwoWords() {
 		assertFalse(dialog.wholeWordCheckBox.getEnabled());
 	}
 
+	@Test
+	public void testActivateDialogWithSelectionActive() {
+		openTextViewer("text" + System.lineSeparator() + "text" + System.lineSeparator() + "text");
+		fTextViewer.setSelectedRange(5, 9);
+		openFindReplaceDialogForTextViewer(false);
+
+		assertFalse(dialog.globalRadioButton.getSelection());
+		dialog.findCombo.setText("text");
+		select(dialog.replaceAllButton);
+		runEventQueue();
+
+		assertEquals(fTextViewer.getDocument().get(), "text" + System.lineSeparator() + System.lineSeparator());
```

Comment:
Please switch expected and actual.

Reply:
I have chosen to use Hamcrest-Matchers instead in order to avoid this confusion. I could imagine this being a problem because this is the only place I use hamcrest in these tests. Since the new FindReplaceDialogTest (introduced in the Find/Replace-Overlay-PR) uses Hamcrest, I don't believe this is a problem.